app/policies/user_policy.rb

```diff
@@ -0,0 +1,14 @@
+class UserPolicy
+  attr_reader :current_user, :user
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+  def show?
```

Comment:
you should probably adjust your logic or add methods in this policy so that users can only access their own feed.

Reply:
...and discover pages.

Reply:
done i believe